observability/otel/otel-rn.adoc

```diff
@@ -8,8 +8,77 @@ toc::[]
 
 include::modules/otel-product-overview.adoc[leveloffset=+1]
 
+You can use the {OTELName} xref:otel-forwarding.adoc#otel-forwarding-traces[in combination with] the xref:../distr_tracing/distr-tracing-rn.adoc#distr-tracing-rn[{TempoName}].
+
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
+[id="otel_3-3_{context}"]
+== Release notes for {OTELName} 3.3
+
+The {OTELName} is provided through the {OTELOperator}.
+
+The {OTELName} 3.3 is based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.107.0.
+
+[id="otel_3-3_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2024-6104[CVE-2024-6104]
+* link:https://access.redhat.com/security/cve/CVE-2024-42368[CVE-2024-42368]
+
+[id="otel_3-3_technology-preview-features_{context}"]
+=== Technology Preview features
+
+This update introduces the following Technology Preview features:
+
+* Group-by-attribute processor
+* Metrics transform processor
+* Routing connector
+* Exporting logs to the OpenShift LokiStack log store
```

Comment:
Do you need OpenShift here? I don't see _OpenShift LokiStack log store_ elsewhere in the docs.

```suggestion
* Group-by-attribute processor
* Metrics transform processor
* Routing connector
* Exporting logs to the LokiStack log store
```

Reply:
maybe we can say to the OpenShift log store base on LokiStack

Reply:
@pavolloffay, I'd like to drop `OpenShift` from its name. We are not supposed to use the word `OpenShift` in the docs. In any case, OpenShift is the context. So it should be clear to say `LokiStack log store`. :slightly_smiling_face: Is this acceptable to you?